storage/src/tests/storageserver/documentapiconvertertest.cpp

```diff
@@ -202,7 +203,7 @@ TEST_F(DocumentApiConverterTest, create_visitor) {
 
 TEST_F(DocumentApiConverterTest, create_visitor_high_timeout) {
     documentapi::CreateVisitorMessage cv("mylib", "myinstance", "control-dest", "data-dest");
-    cv.setTimeRemaining((uint64_t)std::numeric_limits<uint32_t>::max() + 1); // Will be INT_MAX
+    cv.setTimeRemaining(std::chrono::milliseconds(std::numeric_limits<uint32_t>::max() + 1l)); // Will be INT_MAX
```

Comment:
would 'int64_t(1) << 32' be easier to understand (if that is even the number we are making...) ? also, I do not understand the comment.

Reply:
Agree, fixed